### PR TITLE
CASMINST-7088 add note to CSM 1.5 to specify USS key values needed during IUF deliver product

### PR DESCRIPTION
**Pull request by leliasen-hpe**
# Description

We need to make it more clear that the following is necessary in `site_vars.yaml` when installing USS 1.1 or higher.

```text
 uss:
    deploy_slurm: true
    deploy_pbs: false
```

If customers do not do this, they will need to restart IUF at an early stage so we should make it more clear.

# Checklist

- [x] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [x] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [x] My commits or Pull-Request Title contain my JIRA information, or I do not have a JIRA.

 
[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: operations/iuf/workflows/admin_directory.md [hunk 1]
```diff
@@ -69,6 +69,21 @@ the HPC CSM Software Recipe with the existing content in `${ADMIN_DIR}`.
...
+
+       ```text
+       default:
+        network_type: "cassini"
+        suffix: "-test01"
+        system_name: "my-system"
+        site_domain: "my-site-domain.net"
+       uss:
+        deploy_slurm: true
+        deploy_pbs: true
```
Comment (dgloe-hpe) [inline operations/iuf/workflows/admin_directory.md]: Customers will want to install either 0 or 1 WLM, not both. The example should either have false for both of them, or true for only Slurm (it's more popular).

File: operations/iuf/workflows/admin_directory.md [hunk 2]
```diff
@@ -69,6 +69,21 @@ the HPC CSM Software Recipe with the existing content in `${ADMIN_DIR}`.
          suffix: "-test01"
        ```
 
+       **NOTE:** When installing USS 1.1 or higher, select either SLURM or PBS Pro Products to use on the system before running this stage. For more information, see the `deliver-product` stage
```
Comment (dgloe-hpe) [inline operations/iuf/workflows/admin_directory.md]: Slurm, not SLURM

File: operations/iuf/workflows/product_delivery.md [hunk 3]
```diff
@@ -61,8 +61,8 @@ Refer to that table and any corresponding product documents before continuing to
    Additional arguments are available to control the behavior of the `deliver-product` stage (for example, `-rv`). See the [`deliver-product` stage documentation](../stages/deliver_product.md)
    for details and adjust the example below if necessary.
 
-     **`NOTE`** When installing USS 1.1 or higher, select either SLURM or PBS Pro Products to use on the system before running this stage. For more information, see the `deliver-product` stage
-     details in the "Install and Upgrade Framework" section of the _HPE Cray Supercomputing User Services Software Administration Guide: CSM on HPE Cray Supercomputing EX Systems (S-8063)_.
+     **`NOTE`** When installing USS 1.1 or higher, select either SLURM or PBS Pro Products to use on the system before running this stage. This should be specified in `site_vars.yaml`.
```
Comment (dgloe-hpe) [inline operations/iuf/workflows/product_delivery.md]: Slurm, not SLURM

Comment (leliasen-hpe) [reply operations/iuf/workflows/product_delivery.md]: fixed, thanks

File: operations/iuf/workflows/admin_directory.md (hunk 1 above)
Comment (leliasen-hpe) [reply operations/iuf/workflows/admin_directory.md]: fixed, thanks

**Review by don-bahls-hpe — APPROVED** [review]

**Review by dgloe-hpe — APPROVED** [review]

**Review by nrockershousen — APPROVED** [review]

File: operations/iuf/workflows/admin_directory.md [hunk 4]
```diff
@@ -69,6 +69,19 @@ the HPC CSM Software Recipe with the existing content in `${ADMIN_DIR}`.
          suffix: "-test01"
        ```
 
+       **NOTE:** When installing USS 1.1 or higher, select either Slurm or PBS Pro Products to use on the system before running this stage. For more information, see the `deliver-product` stage
+       details in the "Install and Upgrade Framework" section of the _HPE Cray Supercomputing User Services Software Administration Guide: CSM on HPE Cray Supercomputing EX Systems (S-8063)_.
+       If installing USS 1.1 or higher, the `deploy_slurm` and `deploy_pbs` keys should be in the `site_vars.yaml` file as shown below.
+
+       ```text
```
Comment (rustydb) [inline operations/iuf/workflows/admin_directory.md]:
```suggestion
       ```yaml
```

**Review by rustydb — APPROVED** [review]

**Pull request merged**